### ISSUE TITLE
refactor the openshift data usage scripts so it's easier to review/audit

### ISSUE THREAD

**Issue by naved001** [CCI-MOC/ops-issues#1054]
will do it sometime next week.

**Comment by naved001** [comment]
I have been doing this work here: https://github.com/naved001/xdmod-openshift-scripts/tree/dev

* Got rid of most of xdmod related fields from the pod based reports.
* I have linted the files with `black` and addressed a bunch of stuff pointed out by pylint.
* Simplified some function arguments
* I experimented with various intervals for collecting the metrics, and have settled on at collecting metrics every 15 minutes instead every minute. This significantly reduces the size of metrics collected and makes processing/writing the data faster without affecting the accuracy of the report. The metrics that we are collecting don't change often (if cpu, memory, or gpu request is changed, a new pod is launched and the old one killed); but sometimes pods are in some non-running state where they report 0 for those metrics so it's important to track it at some interval.
* If we ever wanted to track actual usage instead of requested resources, then we would want to be collecting metrics at the higher frequency.
* I am using python's csvwriter instead of directly writing to files to simplify things.
